mParticle-Apple-SDK/Kits/MPKitContainer.mm

```diff
@@ -241,6 +242,25 @@ - (void)freeKit:(NSNumber *)integrationId {
     }
 }
 
+- (void)registerMParticleKit {
```

Comment:
registerMParticleKit means register any class implementing the MParticleKitProtocol or specifically the mP kit.
I thing that having an mP kit should give us the ability treat them exactly the same as interchangeable, therefore I would use agnostic names (which in this case I dont understand if we are referring to a concrete implementation or the abstraction)

Reply:
In this case it refers to registering the actual MParticleKit class. My goal with having a separate registration method was to keep it distinct from user provided sideloaded kits. Later we may have other hard coded kits that would also have their own register methods, and this method may be optionally called instead of always called.